### PR TITLE
UofT-DSI | LCR - Assignment 1

### PR DESCRIPTION
**Pull request by postatum**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Completed assignment 1

## What did you learn from the changes you have made?
I learned how to standardize the data, split it into training and test sets, tune KNN classifier, do grid search using different n_neighbors params and folds, predict labels and evaluate prediction accuracy.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I was thinking of two ways to check model accuracy - using KNN classifier method and sklearn utility function.

I also updated data standardization cell so I wouldn't have to bind class back into df. 

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
No, it was fine.

## How were these changes tested?
Made sure they work locally and produce expected results.

## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by fyellow — COMMENTED** [review]
Reviewed!
